Makefile

```diff
@@ -53,6 +53,11 @@ test:
 	PLATFORM=nrf52 cargo test --workspace
 	make examples
 
+.PHONY: analyse-stack-sizes
+analyse-stack-sizes:
+	cargo install stack-sizes
```

Comment:
I don't think we should be unconditionally running `cargo install`. Can you add a check and only run `cargo install` if stack-sizes isn't present?

Reply:
Is there a way to check? Cargo will nicely just not install it if it's already installed.

I can move this to the setup step instead if that is preferred?

Reply:
I would prefer to do the install in the setup step.

Reply:
Updated!